### PR TITLE
Hide Wazuh breadcrumb label and Wazuh api selector css fix

### LINKED ISSUE #3343: Wazuh Menu render issues
| Wazuh | Elastic |
| ----- | ------- | 
| 4.2   | 7.12.1    | 

**Description**
On elastic version 7.12, the wazuh menu bar does not render correctly.
The wazuh logo appears side by side with a small "Wazuh" text, and the API/Index pattern selector is justified to the left instead of the right

**Steps to reproduce**

1. Launch a 4.2-7.12.1 enviroment with either basic or xpack security
2. Go to the app
3. See error

**Screenshots**

### PR DESCRIPTION
**Pull request by asteriscos**
Hi team,

this fixed wazuh main menu render issues in 7.12

Closes #3343 

To test try to resproduce the linked issue

image

### CODE REVIEW

File: public/utils/wz-logo-menu.js [hunk 1]
```diff
@@ -13,7 +13,7 @@
 // Remove Kibana Wazuh name and breadcrumb
 export const changeWazuhNavLogo = () => {
   const interval = setInterval(() => {
-    const nav = $('.euiHeader > .euiBreadcrumbs  > .euiBreadcrumb');
+    const nav = $('[data-test-subj="breadcrumbs"]  > .euiBreadcrumb');
```
Comment (matiasmoreno876) [inline public/utils/wz-logo-menu.js:16]: This is jquery? Wouldn't it be better to use vanilla js? and cut with the mix of technologies?

**Review by frankeros — APPROVED** [review]
LGTM!

**Review by pablomarga — APPROVED** [review]
CR: :heavy_check_mark:

**Review by matiasmoreno876 — APPROVED** [review]
LGTM

**Comment by matiasmoreno876** [comment]
Tested ✅ 

image

**Pull request merged**